### PR TITLE
Absolute tolerance for assert_bin_values

### PR DESCRIPTION
**Pull request by nkorinek**
Hey @lwasser I added in changes to make the docs run on @ryla5068 's PR. Hopefully this is good to merge after this!

addresses #157

### CODE REVIEW

**Comment by nkorinek** [comment]
Sorry to open up a new PR. Made it easier for me to add the changes in.

File: matplotcheck/tests/test_base_data.py [hunk 1]
```diff
@@ -351,28 +352,26 @@ def test_assert_bin_values_incorrect(pt_hist_overlapping):
 
 
 def test_assert_bin_values_tolerance(pt_hist_overlapping):
-    """Test that assert_bin_values correctly passes when using tolerance
-    flag."""
+    """Test that assert_bin_values correctly passes when using tolerence."""
```
Comment (lwasser) [inline matplotcheck/tests/test_base_data.py]:
```suggestion
    """Test that assert_bin_values correctly passes when using tolerance."""
```

File: matplotcheck/tests/test_base_data.py [hunk 2]
```diff
@@ -351,28 +352,26 @@ def test_assert_bin_values_incorrect(pt_hist_overlapping):
...
-    pt_hist_overlapping.assert_bin_values(bin_values, tolerance=0.11)
+    pt_hist_overlapping.assert_bin_values(bin_values, tolerance=2)
 
     plt.close()
 
 
 def test_assert_bin_values_tolerance_fails(pt_hist_overlapping):
-    """Test that assert_bin_values correctly fails when using tolerance
-    flag."""
+    """Test that assert_bin_values correctly fails when using tolerence."""
```
Comment (lwasser) [inline matplotcheck/tests/test_base_data.py]:
```suggestion
    """Test that assert_bin_values correctly fails when using tolerance."""
```

**Comment by lwasser** [comment]
i've made a few spelling corrections @nkorinek  this can be merged once CI is happy again. THANK YOU!!

**Review by lwasser — APPROVED** [review]

**Comment by nkorinek** [comment]
@lwasser passed!

**Comment by lwasser** [comment]
yay merging. thanks @nkorinek !!

**Pull request merged**